tracer/test/test-applications/debugger/Samples.Probes/rcm_config.json

```diff
@@ -0,0 +1 @@
+{"targets":"eyJzaWduZWQiOnsidGFyZ2V0cyI6eyJkYXRhZG9nLzIvTElWRV9ERUJVR0dJTkcvYjYyYjkxMGQtZmFjNS0zZjdjLTlkY2UtZDNhMGZlMjUyYTU5L2NvbmZpZyI6eyJjdXN0b20iOnsidiI6MH0sImhhc2hlcyI6e30sImxlbmd0aCI6MH19LCJ2ZXJzaW9uIjowLCJDdXN0b20iOm51bGx9fQ==","client_configs":["datadog/2/LIVE_DEBUGGING/b62b910d-fac5-3f7c-9dce-d3a0fe252a59/config"],"target_files":[{"path":"datadog/2/LIVE_DEBUGGING/b62b910d-fac5-3f7c-9dce-d3a0fe252a59/config","raw":"eyJJZCI6IjAwMDAwMDAwLTAwMDAtMDAwMC0wMDAwLTAwMDAwMDAwMDAwMCIsIlNuYXBzaG90UHJvYmVzIjpbeyJDYXB0dXJlIjpudWxsLCJTYW1wbGluZyI6bnVsbCwiTGFuZ3VhZ2UiOiJkb3RuZXQiLCJJZCI6IjgyODZkMDQ2LTk3NDAtYTNlNC05NWNmLWZmNDY2OTljNzNjNCIsIk9yZ0lkIjpudWxsLCJBcHBJZCI6bnVsbCwiQWN0aXZlIjp0cnVlLCJUYWdzIjpudWxsLCJXaGVyZSI6eyJUeXBlTmFtZSI6IlNhbXBsZXMuUHJvYmVzLlNtb2tlVGVzdHMuSGFzQXJndW1lbnRzQW5kTG9jYWxzIiwiTWV0aG9kTmFtZSI6Ik1ldGhvZCIsIlNvdXJjZUZpbGUiOm51bGwsIlNpZ25hdHVyZSI6IlN5c3RlbS5TdHJpbmcsU3lzdGVtLlN0cmluZyIsIkxpbmVzIjpudWxsfSwiQWRkaXRpb25hbElkcyI6bnVsbCwiVmVyc2lvbiI6bnVsbH1dLCJNZXRyaWNQcm9iZXMiOltdLCJBbGxvd0xpc3QiOm51bGwsIkRlbnlMaXN0IjpudWxsLCJTYW1wbGluZyI6bnVsbCwiT3BzQ29uZmlndXJhdGlvbiI6bnVsbH0="}]}
```

Comment:
I think this file and the one in `Samples.Security.AspNetCore5` are not needed and left over from the old way of simulating remote config.

Reply:
Fixed in 7cfdd0b5119010410961a6482681af0dbcde1d09